pages/index.tsx

```diff
@@ -1,62 +1,54 @@
-import fs from 'fs'
-import path from 'path'
-
-import { useContext } from 'react'
-
-import matter from 'gray-matter'
 import type { NextPage } from 'next'
-import { serialize } from 'next-mdx-remote/serialize'
-import getConfig from 'next/config'
 import Head from 'next/head'
-import { IItemDocs, IGasDocs, IDocMeta } from 'types'
-
-import { EthereumContext } from 'context/ethereumContext'
 
-import ContributeBox from 'components/ContributeBox'
+import Editor from 'components/Editor'
 import HomeLayout from 'components/layouts/Home'
-import ReferenceTable from 'components/Reference'
-import { H1, Container } from 'components/ui'
-
-const { serverRuntimeConfig } = getConfig()
-
-const HomePage = ({
-  opcodeDocs,
-  gasDocs,
-}: {
-  opcodeDocs: IItemDocs
-  gasDocs: IGasDocs
-}) => {
-  const { opcodes } = useContext(EthereumContext)
+import { Container } from 'components/ui'
 
+const HomePage = () => {
   return (
     <>
+      <html lang="en"></html>
       <Head>
         <meta property="og:type" content="website" />
-        <meta property="og:title" content="EVM Codes - Opcodes" />
+        <title>Kakarot Playground - Playground</title>
         <meta
           name="description"
-          content="An Ethereum Virtual Machine Opcodes Interactive Reference"
+          content="Kakarot Playground’ Playground is an interactive tool for learning how to use opcodes, the stack behind smart contracts."
```

Comment:
is it a stack?
maybe `[...] to use opcodes, the instructions behind smart contracts`

Reply:
Ah c'est quand j'ai interverti la page opcodes et la page playground mais c'est le texte initial de evm.codes je l'ai pas touché pour le coup